src/components/WalletPasswordStep/WalletPasswordStepContainer.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import { pure } from 'recompose'
+
+import WalletPasswordStep from './WalletPasswordStep'
+
+export default pure(WalletPasswordStep)
```

Comment:
Here, there will be problems with flowtype. All prop types will be unavailable in another files.
After discussion we decided to use React.PureComponent instead of recompose/pure

Reply:
https://github.com/jibrelnetwork/jwallet-web/tree/feature/update-import-wallet/src/components/WalletPasswordStep

Reply:
just functional component